packages/engine/src/scene/functions/SceneLoading.ts

```diff
@@ -192,24 +178,19 @@ export class WorldScene {
         createGround(entity, component.data)
         break;
 
-      case 'skybox':
-        createSkybox(entity, component.data);
-        break;
-
       case 'image':
         addObject3DComponent(entity, { obj3d: Image, objArgs: component.data });
         break;
 
       case 'video':
         // if livestream, server will send the video info to the client
         if (isClient) {
-          // if(!component.data.isLivestream) {
-          createVideo(entity, component.data);
-          // }
-          // addComponent(entity, LivestreamComponent)
-          // } else if(component.data.isLivestream) {
-          // @todo
-          // addComponent(entity, LivestreamProxyComponent, { src: component.data.src })
+          if(!component.data.isLivestream) {
+            createVideo(entity, component.data);
+          }
+          addComponent(entity, LivestreamComponent)
```

Comment:
livestream things can be ignored for now, they are not fully implemented and can cause problems